doc/classes/ModStorage.adoc

```diff
@@ -0,0 +1,57 @@
+= ModStorage
+
+include::../include/config.adoc[]
+include::../include/types.adoc[]
+:description: Per-mod persistent key-value store
+:keywords: meta, data, metadata, mod, storage, key, value, persistence
+
+ModStorage is a per-world, per-mod persistent string key-value store implementing all methods of MetaData.
+
+The granularity of the persisted snapshots is determined by the `map_save_interval` setting.
+
+WARNING: Incapable of saving raw binary data due to JSON restrictions.
```

Comment:
this is incorrect for the new sql backed version https://github.com/minetest/minetest/issues/12167

Reply:
I'm aware, but modders may not rely on the SQLite backend being used.

Reply:
correct, but it should be noted that it should not be done for that reason

Reply:
https://github.com/minetest/minetest_docs/pull/40/commits/b468f41588077ef6a48d63bfa51b3c7fe61d9ec1